modules/apps/29-fee/keeper/escrow.go

```diff
@@ -0,0 +1,106 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/cosmos/ibc-go/modules/apps/29-fee/types"
+	channeltypes "github.com/cosmos/ibc-go/modules/core/04-channel/types"
+)
+
+// EscrowPacketFee sends the packet fee to the 29-fee module account to hold in escrow
+func (k Keeper) EscrowPacketFee(ctx sdk.Context, refundAcc sdk.AccAddress, identifiedFee *types.IdentifiedPacketFee) error {
+	// check if the refund account exists
+	hasRefundAcc := k.authKeeper.GetAccount(ctx, refundAcc)
+	if hasRefundAcc == nil {
+		return sdkerrors.Wrap(types.ErrRefundAccNotFound, fmt.Sprintf("Account with address: %s not found", refundAcc.String()))
+	}
+
+	coins := identifiedFee.Fee.AckFee
+	coins = coins.Add(identifiedFee.Fee.ReceiveFee...)
+	coins = coins.Add(identifiedFee.Fee.TimeoutFee...)
```

Comment:
can't you just used `coins := identifiedFee.Fee`?

Reply:
`identifiedFee.Fee` is not of type` sdk.Coins` it is three separate `sdk.Coins`